app/views/repositories/index.html.erb

```diff
@@ -3,27 +3,46 @@
 
 <% if current_team %>
   <%= render partial: "sidebar", locals: { repositories: @repositories } %>
-  <%
-    # show only if no repositories present. If the team will have them we will
-    # handle this in left side navigation bar
-  %>
-  <div class="content-pane">
-    <div class="jumbotron text-center" style="margin-top:12%">
-      <strong><%=t 'libraries.index.no_libraries.text' %></strong>
+  <div class="content-pane flexible">
+    <div class="content-header">
+      <h1><%= t('libraries.index.head_title') %></h1>
+    </div>
+    <div class="content-body">
+    </div>
+    <template id="activeRepositoriesListTable">
```

Comment:
Should we check how `template` is doin with IE? 
Here is marked as not supported
https://developer.mozilla.org/en-US/docs/Web/HTML/Element/template

Reply:
it supported partially, it will be moved out from body. But anyway SciNote not correctly working in IE.